### PR TITLE
I've added archived task view page. With the button on the main task …

### PR DESCRIPTION
**Pull request by rgossomTechtonic**
…page that links to the ArchivedTaskView page.

                      °The only problem I'm having issues with is removing the edit and delete button°.
                                                °We might have to make that a separate ticket°

### CODE REVIEW

**Comment by umorenovanderlaanTechtonic** [comment]
Ayyy, well done! I'll throw that into Jira. We'll probably have to approach it by having a separate render- instead of using lower level tasks, we'd have an archived task list that would render anything with completed:true differently.

**Comment by rgossomTechtonic** [comment]
Ok, sweet sounds good to me.

File: src/components/navBar/NavBar.js [hunk 1]
```diff
@@ -21,6 +22,12 @@ class NavBar extends Component {
     });
   };
 
+  taskView() {
```
Comment (jhernandezinzunzaTechtonic) [inline src/components/navBar/NavBar.js]: I'm not too sure what this is doing, it seems that the button is wrapped in an anchor tag which takes us to the archive, so the onclick of the button isn't doing anything.

**Review by jhernandezinzunzaTechtonic — DISMISSED** [review]

**Comment by jhernandezinzunzaTechtonic** [comment]
Looks awesome minus that extra empty function, nice job.  We can build onto that as we continue.

File: src/components/navBar/NavBar.js (hunk 1 above)
Comment (rgossomTechtonic) [reply src/components/navBar/NavBar.js]: Ok, I'll look at it now : (

**Comment by rgossomTechtonic** [comment]
I have removed the extra function so it should be good to merge now.

**Pull request merged**